.drone.yml

```diff
@@ -35,66 +35,25 @@ steps:
     group: build
     image: golang
     pull: always
-    depends_on: 
+    depends_on:
       - lint
     environment:
       GOOS: darwin
       GOARCH: amd64
-      CGO_ENABLED: 0    
-    commands:
-      - go mod vendor
-      - go build 
-    when:
-      event:
-        - push
-
-  - name: build-linux
-    group: build-to-release
-    image: golang
-    pull: always
-    depends_on: 
-      - lint
-    environment:
-      GOOS: linux
-      GOARCH: amd64
       CGO_ENABLED: 0
     commands:
       - go mod vendor
-      - go build -a -ldflags '-extldflags "-static"' -o bin/releases/furyagent-linux-amd64  .
+      - go build
     when:
       event:
-        - tag
+        - push
 
-  - name: build-darwin
-    group: build-to-release
+  - name: build-release-homebrew
```

Comment:
build-release instead of build-release-homebrew as it releases also to github releases page ;)